### PR TITLE
Update ruRu

### PR DESCRIPTION
**Pull request by IKnewOne**

### CODE REVIEW

File: Aberrus/Locales/ruRU.lua [hunk 1]
```diff
@@ -38,19 +38,19 @@ L = BigWigs:NewBossLocale("Assault of the Zaqali", "ruRU")
 if L then
 	-- These are in-game emotes and need to match the text shown in-game
 	-- You should also replace the comment (--) with the full emote as it shows in-game
-	--L.zaqali_aide_north_emote_trigger = "northern battlement" -- |TInterface\\ICONS\\Ability_Hunter_KillCommand.blp:20|t Commanders ascend the northern battlement!
-	--L.zaqali_aide_south_emote_trigger = "southern battlement" -- |TInterface\\ICONS\\Ability_Hunter_KillCommand.blp:20|t Commanders ascend the southern battlement!
+	L.zaqali_aide_north_emote_trigger = "южное укрепление" -- |TInterface\\ICONS\\Ability_Hunter_KillCommand.blp:20|t Commanders ascend the northern battlement!
```
Comment (funkydude) [inline Aberrus/Locales/ruRU.lua:41]: As stated in the comments above these lines, you need to replace the comment with the full translation from in-game so we have a reference if we need to change it in future.

File: Aberrus/Locales/ruRU.lua [hunk 2]
```diff
@@ -38,19 +38,19 @@ L = BigWigs:NewBossLocale("Assault of the Zaqali", "ruRU")
...
-	--L.both = "Both"
+	L.north = "Север"
+	L.south = "Юг"
+	L.both = "Обе"
 
-	--L.zaqali_aide_message = "%s Climbing %s" -- Big Adds Climbing North
+	L.zaqali_aide_message = "%s лезут:  %s" -- Big Adds Climbing North
 	L.add_bartext = "%s: %s (%d)"
-	--L.boss_returns = "Boss Lands: North"
+	L.boss_returns = "Босс: North"
```
Comment (funkydude) [inline Aberrus/Locales/ruRU.lua:50]: You missed translating "North" here

**Comment by funkydude** [comment]
Some comments above that need addressed

**Comment by funkydude** [comment]
I think you're misunderstanding, the comment is this part `-- |TInterface\\ICONS\\Ability_Hunter_KillCommand.blp:20|t Commanders ascend the southern battlement!`

**Comment by IKnewOne** [comment]
Reading comprehensions is difficult.
Also didn't tag that PR intentionally

**Comment by funkydude** [comment]
Don't worry about it :) and thank you!
According to google translate `южное укрепление` is south but you have it set for the north trigger?

**Pull request merged**